app/src/main/java/com/kamron/pogoiv/activities/ClipboardModifierParentFragment.java

```diff
@@ -59,6 +59,21 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         return view;
     }
 
+    @Override
+    //Pressed return button - in the case when coming back from settings screen after movesets have been enable/disabled
+    public void onResume() {
```

Comment:
Avoid all this behavior, all this added complexity just to remove the moveset clipboard tokens from the list is overkill.
If a user have deactivated the moveset functionality in the settings, he would probably just ignore the movesets tokens knowing that they won't be effective.
Also, moveset tokens should already work when the movesets info isn't present (i.e. when the moveset scan fails).